.buildkite/pipeline.smoke.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+  - label: "Smoke Tests: Catalogue API (stage)"
+    plugins:
+      - wellcomecollection/aws-assume-role#v0.2.2:
+          role: "arn:aws:iam::130871440101:role/experience-ci"
```

Comment:
Is there a reason for this to be `experience-ci` over `catalogue-ci`?

Reply:
The secret is in the experience account, so in order to get it without specifying the whole ARN of the secret I'm assuming the experience role. I'm not sure how correct this is though - i'll have a think about it.